content/docs/resiliency/_index.md

```diff
@@ -52,7 +52,7 @@ CSM for Resiliency provides the following capabilities:
 {{<table "table table-striped table-bordered table-sm">}}
 |               | PowerFlex    | Unity XT                          | PowerScale                              | PowerStore                    |
 | ------------- | :----------: | :-------------------------------: | :-------------------------------------: | :---------------------------: |
-| Storage Array | 3.5.x, 3.6.x | 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2, 5.2, 5.3 | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4 | 1.0.x, 2.0.x, 2.1.x, 3.0, 3.2 |
+| Storage Array | 3.5.x, 3.6.x, 4.0.x, 4.5 | 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2, 5.2, 5.3 | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4 | 1.0.x, 2.0.x, 2.1.x, 3.0, 3.2 |
```

Comment:
Please remove 3.5.x, 3.6.x for PowerFlex
Replace OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 with OneFS 9.3, 9.4, 9.5.0.5, 9.5.0.6
Replace Unity: 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2, 5.2, 5.3 with 5.1.2, 5.2, 5.3

Reply:
> Please remove 3.5.x, 3.6.x for PowerFlex Replace OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 with OneFS 9.3, 9.4, 9.5.0.5, 9.5.0.6 Replace Unity: 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2, 5.2, 5.3 with 5.1.2, 5.2, 5.3

Fixed